esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -763,6 +763,9 @@ define(
     Year: "Año",
     YouAreAboutToDeleteBinding: "Está a punto de eliminar esta conexión. Desea continuar?",
     YouAreAboutToRemoveUserFrom: "Usted esta a punto de quitar un(os) usuario(s) de este grupo. ¿Desea continuar?",
+    YouAreAboutToDeleteThisFile: "Usted esta a punto de eliminar este archivo. ¿Desea continuar?",
+    YouAreAboutToDeleteThisQueryset: "Usted esta a punto de eliminar este grupo de query. ¿Desea continuar?",
```

Comment:
Looked through the file and for the most part we do translate query to consulta. Although I did see a few entries where we don't translate query.
And perhaps outside the scope of this issue, but there's a instace where we (prob me) misspelled Consulta:
ResetThisQuery: "Reajustar esta Consulat",